src/collection/dimensions.js

```diff
@@ -266,6 +266,20 @@ fn = elesfn = ({
         };
       }
 
+      function computePaddingValues( width, paddingObject ) {
```

Comment:
Ok. I would prefer the idea of having `padding-relative-to: width | height | min | max | average` though. I tried to follow the spec here (https://www.w3.org/TR/CSS2/box.html#padding-properties) so that padding behaves like people are used to, which means everything is calculated relative to width. I think if it behaves any other way, it wouldn't really be padding. We can maybe make padding an alias for `padding-relative-to: width` for backwards compatibility.